encryption_works.md

```diff
@@ -236,23 +242,25 @@ On the other end, the receiver uses PGP to decrypt the message and verify the e-
 
-Like OTR, each PGP key has a unique fingerprint. [Here's](https://pgp.mit.edu/pks/lookup?op=get&search=0xE7E8E7D097604F9D) the PGP public for Tommy Collison, one of the authors of this guide. That key's fingerprint is 696E C53E 8535 6DE8 10C3 75D2 E7E8 E7D0 9760 4F9D. If you look at my public key, you'll see that it's long and would be difficult to verbally confirm with another person. A fingerprint is a short and more convenient way to uniquely represent a key.
+![A sample PGP block.](images/pgp_block.png)
 
-In 2013, Kevin Poulsen [published](http://www.wired.com/threatlevel/2013/06/signed-bda0df3c/) a PGP encrypted message on Wired's website intended for Edward Snowden to read. As long as Wired has a copy of Snowden's real public key, only someone in possession of Snowden's secret key can decrypt this message.
+Like OTR, each PGP key has a unique fingerprint. [Here's](https://pgp.mit.edu/pks/lookup?op=get&search=0xE7E8E7D097604F9D) the PGP public for Tommy Collison, one of the editors of this guide. That key's fingerprint is 696E C53E 8535 6DE8 10C3 75D2 E7E8 E7D0 9760 4F9D. Look at the PGP block above -- it's long and would be difficult to verbally confirm with another person. A fingerprint is a short and more convenient way to uniquely represent a key.
 
-You can think of your public key and private key like your e-mail address and e-mail passphrase. You probably want your general e-mail address to be public, so that people can contact you -- it's the same with your public key. And, in the same way you should keep your e-mail passphrase to yourself, you should never share your private key with anyone or attach it in an e-mail.
+You can think of your public key and private key like your e-mail address and e-mail passphrase. You probably want your general e-mail address to be public, so that people can contact you -- it's the same with your public key. And, in the same way you should always keep your e-mail passphrase to yourself, you should never share your private key with anyone or attach it in an e-mail.
 
-It's a good idea to publicize your public key, especially if you create a new pair. Many people [tweet](https://twitter.com/tommycollison/status/620696207187398656) their key's fingerprint, [mention](http://parkerhiggins.net/2014/08/email-signature-nudge-encryption-use/) it in their email signatures, or [link](https://twitter.com/ggreenwald) to it in their Twitter bio:
+It's a good idea to publicize your public key, so that people who want to contact you can be sure that the public key they use belongs to you. Many people [tweet](https://twitter.com/tommycollison/status/620696207187398656) their key's fingerprint, [mention](http://parkerhiggins.net/2014/08/email-signature-nudge-encryption-use/) it in their email signatures, or [link](https://twitter.com/ggreenwald) to it in their Twitter bio:
```

Comment:
This is pedantic, but it might be better to say "to help people who want to contact you verify that the public key they use belongs to you", instead of "be sure".

A PGP key/link/fingerprint is only as good as the source it was obtained from. Getting it from Twitter is only good as long as the access to Twitter was over HTTPS, you trust Twitter, Twitter hasn't been hacked, that TLS is secure, that there are no rogue CA's operating at the behest of or compromised by your adversary, etc... Same is true of every other method for sharing this info online. For some people this is "good enough". Others like to share the fingerprint as widely as possible, *on as many different platforms as possible*, under the assumption that a careful user will check multiple locations and it would be hard for an adversary to spoof them all.

If you're really paranoid, the only way to go is to meet the person and sign their key yourself, or use the WoT. I understand that the WoT has its own section later, I just don't want to give people the impression that verifying someone's fingerprint against what's on their website, in their email or signature, or in their Twitter bio is necessarily sufficient.